lib/mina_sidekiq/tasks.rb

```diff
@@ -33,7 +33,7 @@
 
 # ### sidekiqctl
 # Sets the path to sidekiqctl.
-set :sidekiqctl, -> { "#{fetch(:bundle_prefix)} sidekiqctl" }
+set :sidekiqctl, -> { "#{fetch(:bundle_bin)} exec sidekiqctl" }
```

Comment:
With this change, `RAILS_ENV` wouldn't be explicitly set.

```
:bundle_bin             #=> 'bundle'
:bundle_prefix          #=> %{RAILS_ENV="#{fetch(:rails_env)}" #{fetch(:bundle_bin)} exec}
```

Reply:
`sidekiqctl` has two commands only: `quiet` and `stop`. And it does not rely on `RAILS_ENV`, cause both of commands assume that sidekiq has already started in some environment. So it cannot break any code at all.